.github/workflows/publish.yaml

```
@@ -11,3 +11,21 @@ jobs:
     with:
       image-name: network-services-operator
     secrets: inherit
+
+  publish-kustomize-bundles:
+    permissions:
+      id-token: write
+      contents: read
+      packages: write
+    strategy:
+      matrix:
+        bundles:
+          - name: ghcr.io/datum-cloud/network-services-operator/network-services-crds
+            path: config/crd
+          - name: ghcr.io/datum-cloud/network-services-operator/network-services-operator
+            path: config/network-services-operator
```

@OscarLlamas6 you should be able to use the `config/default` program to export the operator's manifests. 

```suggestion
          - name: ghcr.io/datum-cloud/network-services-operator-kustomize
            path: config/default
```

---

@scotwells what do you think about this to have both under */network-services-operator* ?

```css
bundles:
          - name: ghcr.io/datum-cloud/network-services-operator/network-services-crds
            path: config/crd
          - name: ghcr.io/datum-cloud/network-services-operator/network-services-operator-kustomize
            path: config/default
```

---

@scotwells if we have config/default as a kustomization we dont need the **_ghcr.io/datum-cloud/network-services-operator/network-services-crds_** or do we need it? Since the /config/default kustomization includes crds it will install everything right?